ONSdigital/eq-author-app
# Header redesign

### What is the context of this PR?
Header redesign as per prototype.

Interesting changes
1. There are now 2 headers - one defined in EditorLayout and one for the non editor pages in Layout
1. I have completely removed the properties panel from EditorLayout as it is a question page thing and not generic to all pages in the editor.

### How to review 
1. Ensure it matches the prototype `prototype-new-header-nav-2`
1. Ensure that everything works as expected
